ios/Voice/Voice.m

```diff
@@ -224,7 +225,7 @@ - (void) setupAndStartRecognizing:(NSString*)localeStr {
         [mixer installTapOnBus:0 bufferSize:1024 format:recordingFormat block:^(AVAudioPCMBuffer * _Nonnull buffer, AVAudioTime * _Nonnull when) {
             //Volume Level Metering
             //Buffer frame can be reduced, if you need more output values
-            [buffer setFrameLength: buffer.frameCapacity];
+            // [buffer setFrameLength: buffer.frameCapacity];
```

Comment:
Should we make this a prop? so people can customize

Reply:
Do we need to specify frameCapacity?

Reply:
Gotcha, checked the previous commits looks like we don't need it. Lets clean up the code and delete the line & comments